lib/sisimai/bite/email/mxlogic.rb

```diff
@@ -0,0 +1,253 @@
+module Sisimai::Bite::Email
+  module MXLogic
+    # Sisimai::Bite::Email::MXLogic parses a bounce email which created by
+    # McAfee SaaS (formerly MX Logic).
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/MXLogic.pm
+      # Based on Sisimai::Bite::Email::Exim
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'      => %r/\AMail Delivery System/,
+        :'subject'   => %r{(?:
```

Comment:
__[rubocop]__
* Style/SymbolLiteral: Do not use strings for word-like symbol literals.

<!-- Comment by [SideCI](https://sideci.com) -->